doc/Extensions/RRDCached.md

```diff
@@ -124,6 +124,46 @@ OPTS="$OPTS -w 1800 -z 1800 -f 3600 -t 4"
 ```php
 $config['rrdcached'] = "unix:/var/run/rrdcached.sock";
 ```
+### RRDCached installation Debian Stretch (rrdcached 1.6.0)
+
+1. Install rrdcached
+```bash
+sudo apt-get install rrdcached
+```
+
+2. Edit /etc/default/rrdcached to include:
+```bash
+DAEMON=/usr/bin/rrdcached
+WRITE_TIMEOUT=1800
+WRITE_JITTER=1800
+WRITE_THREADS=4
+BASE_PATH=/mnt/librenms/rrd/db
+JOURNAL_PATH=/mnt/librenms/rrd/journal/
+PIDFILE=/var/run/rrdcached.pid
+SOCKFILE=/var/run/rrdcached.sock
+SOCKGROUP=librenms
+DAEMON_GROUP=librenms
+DAEMON_USER=librenms
+BASE_OPTIONS="-B -F -R"
+```
+
+3. Restart the rrdcached service
+```bash
+    systemctl restart rrdcached.service
+```
+
+4. Edit /opt/librenms/config.php to include:
+
+For local RRDCached server
+```php
+$config['rrdtool_version'] = '1.5.5';
+$config['rrdcached'] = "unix:/var/run/rrdcached.sock";
+```
+For remote RRDCached server
+```php
+$config['rrdtool_version'] = '1.5.5';
+$config['rrdcached'] = "0:42217";
```

Comment:
This won't work with the example config above so I'd remove that to avoid confusion.

Reply:
you are right, made some change to add network option in rrdcached to enable networking